.circleci/config.yml

```diff
@@ -55,6 +55,9 @@ jobs:
       - run:
           name: Type Checking
           command: yarn run type-check
```

Comment:
Is this step still required?

We seem to be calling `tsc` in `build:types`: https://github.com/algolia/instantsearch.js/blob/364d8f8291252900f3136acfa56f73960029c81b/scripts/build/types.js#L11

Reply:
it's with the declaration config, which isn't the same as far as I can tell